modules/social_features/social_landing_page/social_landing_page.module

```diff
@@ -237,6 +225,13 @@ function social_landing_page_preprocess_node(&$variables) {
       }
     }
   }
+  // Add no_image flag if there are no image uploaded.
+  $variables['no_image'] = TRUE;
+  $image_field = "field_{$node->getType()}_image";
+
+  if (!empty($node->{$image_field}->entity)) {
+    $variables['no_image'] = FALSE;
+  }
```

Comment:
We already defined `no_image` variable before. But logic was in few places: `socialbase` theme, `social_core.module` and `social_landing.module`. 

I have refactored it, moved all code to `social_core` since we depend on other code used there add a common variable `node_image` for all nodes. 
See commit: 5ae7982f973f7882778aafe9b5090c2ec1615339